packages/roosterjs-editor-core/lib/corePlugins/ImageSelection.ts

```diff
@@ -92,6 +89,17 @@ export default class ImageSelection implements EditorPlugin {
                         }
                     }
                     break;
+                case PluginEventType.ContextMenu:
+                    const contextMenuTarget = event.rawEvent.target;
+                    const actualSelection = this.editor.getSelectionRangeEx();
+                    if (
+                        safeInstanceOf(contextMenuTarget, 'HTMLImageElement') &&
+                        (actualSelection.type !== SelectionRangeTypes.ImageSelection ||
+                            (actualSelection.type === SelectionRangeTypes.ImageSelection &&
```

Comment:
You can remove the check to `actualSelection.type === SelectionRangeTypes.ImageSelection` because when it goes here, this must be true